### PR TITLE
A11y 35 alt text circuit playground images

### PR DESCRIPTION
**Pull request by mgc1194**
Adding alt text to the following images:
image

## Links
- [Maker toolkit setup page](https://studio.code.org/maker/setup?enableExperiments=microbit)
- jira ticket: [A11Y-35](https://codedotorg.atlassian.net/browse/A11Y-35)

## Testing story
Testing links: [http://localhost-studio.code.org:3000/maker/setup?enableExperiments=microbit](http://localhost-studio.code.org:3000/maker/setup?enableExperiments=microbit)
image

Used Screen Reader Chrome extension to test Alt text.  

## PR Checklist:

- [x] Tests provide adequate coverage
- [x] Privacy and Security impacts have been assessed
- [x] Code is well-commented
- [x] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [x] User impact is well-understood and desirable
- [x] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/i18n/applab/en_us.json [hunk 1]
```diff
@@ -361,6 +361,7 @@
...
   "makerSetupChromebookPage": "Open this page ([{makerSetupPage}]({makerSetupPage})) on your Chromebook.",
   "makerSetupCircuitPlaygroundDescription": "The Adafruit [Circuit Playground Express](https://www.adafruit.com/product/3399) (and the older [Circuit Playground Classic](https://www.adafruit.com/product/3000)) is a programmable circuit board with built-in components that make getting started with physical computing quick and fun.\n\nIn the physical computing unit of [Computer Science Discoveries](https://code.org/educate/csd) students use the Circuit Playground tethered via USB to their computers to allow for online curriculum integration and interactive debugging of programs.\n\nPlease follow the instructions below to connect your board to Code.org.",
+  "makerSetupCircuitPlaygroundImageAltText": "The Circuit Playground board is a circular circuit board with a USB connector at the top rim, 14 connection pins along the rim, 10 LED lights, two buttons and multiple sensors.",
```
Comment (hannahbergam) [inline apps/i18n/applab/en_us.json:364]: I think these are so great! Interested in feedback: I'm torn on whether the first part 'The circuit playground board' is necessary here. I wonder if the title header is enough context.

**Review by hannahbergam — APPROVED** [review]
A comment on whether including the title in both descriptions is necessary, but I think I could go either way! LGTM!

File: apps/i18n/applab/en_us.json (hunk 1 above)
Comment (megcrenshaw) [reply apps/i18n/applab/en_us.json:364]: I'm good with keeping it, but don't have a strong opinion either way!

Comment (mgc1194) [reply apps/i18n/applab/en_us.json:364]: I do agree, I think either way would be arbitrary. I am going to keep it as is, as the images and Alt text could be used in other parts of the website with less context.

**Pull request merged**